mkt/reviewers/views.py

```diff
@@ -918,7 +918,7 @@ def _retrieve_translation(text, language):
 
 
 @waffle_switch('reviews-translate')
-@reviewer_required
+@permission_required([('Apps', 'ModerateReview')])
```

Comment:
wouldn't `@reviewer_required(moderator=True)` work and be more consistent with the rest ?